string data
uint8 wait_time
---
bool success
